docs/contributing/gatsby-style-guide.md

```diff
@@ -78,6 +78,30 @@ rounds of proofreading and editing before you're happy with your writing.
 Also, there's a community of contributors to support you. Bounce ideas off of them and ask for input on your writing in the
 [Gatsby Discord](https://gatsby.dev/discord) and in the [GitHub repo](https://github.com/gatsbyjs/gatsby).
 
+### Use the linter
+
+Gatsby uses [`remark-lint`](https://github.com/remarkjs/remark-lint) and [`retext`](https://github.com/retextjs/retext) in order to check for common spelling, grammar, and formatting errors, including several of the suggestions in this guide. You can run the linter by typing the following on the command line:
+
+```shell
+yarn lint:docs
+```
+
+#### Updating the dictionary
+
+We use [`retext-spell`](https://github.com/retextjs/retext-spell) for spell checking the docs. Since the linter doesn't know about proper nouns, such as individual names or brand names, it may list these as errors:
+
+```text
+83:34-83:40  warning  `retext` is misspelt; did you mean `pretext`, `retest`?  retext                      retext-spell
+```
+
+You can add these words to the dictionary with the following command:
```

Comment:
here i would add a hint to check at first the code fences. 

something like this (but it needs better wording):

```markdown
At first check if these words are are variables, component names, 
function names, and packages, then you can put them in 
code fences (see [Code formatting: Inline code](...)). 
Otherwise you can add the remaining words to the dictionary with the following command:
```

Reply:
and also to check the brand names before checking, otherwise wrong spelled brand names get into the dictionary...